packages/components/textarea/src/textarea.scss

```diff
@@ -4,13 +4,17 @@ $variant-selectors: ':not([valid],[invalid])', ':where([valid])', ':where([inval
 $resize-types: none, both, horizontal, vertical, auto;
 
 :host {
+  --sl-textarea-rows: 3;
   --_border-size: var(--sl-border-width-button-default);
   --_border: var(--_border-size) solid var(--_border-color);
   --_border-radius: var(--sl-border-radius-default);
   --_box-shadow: var(--sl-box-shadow-none);
   --_transition: 200ms ease-in-out;
-  --sl-textarea-min-height-md: 72px;
-  --sl-textarea-min-height-lg: 88px;
+  --_min-height: calc(var(--sl-textarea-rows) * 1lh);
+  // TODO: needs to be removed in the future when lh unit will be supported for Firefox as well
+  @supports (-moz-appearance: none) {
+    --_min-height: calc(var(--sl-textarea-rows) * 1.4em);
```

Comment:
isn't there a token that we can use for this? are you sure it's always 1.4em?

Reply:
This is just an approximate value, I hope we will be able to remove it in the future, when the `lh` will be also supported by Firefox.
We have a line-hight and font-size in `px` which come from tokens, I'm not sure if it's possible for now to calculate it fast in css? Or maybe you have a better idea how to implement it in a better way?  Will be appreciated :)

Reply:
Or maybe we should calculate it just with line-height which comes from token, but it's in px?
Sth like:
```
  @supports not (min-height: 1lh) {
    --_min-height: calc(var(--sl-textarea-rows) * var(--sl-text-typeset-line-height-md));
  }
  ```
  and of course `--sl-text-typeset-line-height-lg` for the large version?

Reply:
I've just changed it this way.